<?xml version="1.0" ?>
<sdf version="1.6">
  <model name="Blue_Checker">
    <static>false</static>
    <link name="blue_checker">
      <pose>0 0 0.015 0 0 0</pose>
      <collision name="collision">
        <geometry>
            <cylinder>
              <radius>.02</radius>
              <length>.03</length>
            </cylinder>
        </geometry>
        <surface>
        <friction>
          <ode>
            <mu>0.8</mu>
            <mu2>0.8</mu2>
          </ode>
        </friction>
      </surface>
      </collision>
      <visual name="visual">
        <cast_shadows>false</cast_shadows>
        <geometry>
            <cylinder>
              <radius>.02</radius>
              <length>.03</length>
            </cylinder>
        </geometry>
        <material>
        <script>
            <uri>file://media/materials/scripts/gazebo.material</uri>
            <name>Gazebo/Blue</name>
        </script>
        </material> 
      </visual>
     <inertial>
        <mass>0.021</mass>
        <inertia> 
          <ixx>0.000002</ixx>       
          <ixy>0.0</ixy>         
          <ixz>0.0</ixz>        
          <iyy>0.000002</iyy>       
          <iyz>0.0</iyz>         
          <izz>0.000002</izz>       
        </inertia>
      </inertial>
    </link>
  </model>


</sdf>